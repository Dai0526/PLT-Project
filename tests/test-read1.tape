int main() 
{
  file filea;
  string array;

  array = new[10000];

  filea = open("test2.txt", "r");
  read(array, 1,125,filea);
  print_s(array);

  return 0;
}
