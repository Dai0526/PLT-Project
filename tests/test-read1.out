Testing for read
Second line for read
Third line
end
